app/src/main/scala/org/alephium/explorer/persistence/DBRunner.scala

```diff
@@ -23,7 +23,7 @@ import slick.jdbc.PostgresProfile
 import slick.jdbc.PostgresProfile.api._
 
 trait DBRunner {
-  val databaseConfig: DatabaseConfig[PostgresProfile]
+  def databaseConfig: DatabaseConfig[PostgresProfile]
```

Comment:
shouldn't we have a `lazy val` instead? in case someone is calling twice `databaseConfig`

Reply:
Please correct me if I'm wrong but I don't think traits can have `lazy` definitions. 

If it does then the object or class that implements this trait would define it as `val`, `var`, `lazy val` or `def` and its definition would get used during runtime not the one in the trait.

Reply:
Ah I think you know this stuff already. In summary it shouldn't need a lazy val 🤔

Reply:
ah stupid me, I didn't notice it was only the definition in the trait :face_palm: